queue_services/entity-emailer/src/entity_emailer/email_templates/BC-IA-COMPLETED.html

```diff
@@ -17,17 +17,25 @@
           [[header.html]]
 
           <div class="container">
-            <p>You have successfully registered a business with BC Business Registry.</p>
+            <p>
+              You have successfully registered 
+              {{ 'your Cooperative Association' if business.legalType == 'CP' else 'a business' }} 
+              with BC Business Registry.
+            </p>
 
-            [[whitespace-16px.html]]
             [[business-info.html]]
 
-            [[whitespace-24px.html]]
-            <p>Please find the following documents attached to this email:</p>
+            <p>The following documents attached to this email:</p>
```

Comment:
missing an `are` I believe.  i.e. `<p>The following documents attached to this email:</p>` should be `<p>The following documents are attached to this email:</p>`